### DISCUSSION TITLE
tailwindcss plugins

### DISCUSSION THREAD

**Discussion by lizhuoqi** [jpanther/congo#255] [Q&A]
how to add support for tailwindcss plugins，  like `line-clamp`?

**Comment by jpanther** [comment]
In order to use Tailwind plugins you'll need to install the necessary dependencies, update your Tailwind config file and build the theme manually. You can find details about the process of [building the theme from source](https://jpanther.github.io/congo/docs/advanced-customisation/#building-the-theme-css-from-source) in the documentation.
